lib/libutils/ext/include/util.h

```diff
@@ -28,10 +28,18 @@
 	 _a < _b ? _a : _b; }))
 #else
 #define MAX(a, b)	(((a) > (b)) ? (a) : (b))
-#define MIN(a, b)	(((a) < (b)) ? (a) : (b))
+#define MIN(a, b)	(((a) > (b)) ? (a) : (b))
```

Comment:
Oh!

Reply:
gosh, i fixed that, but pushed to bad one  :(